app/src/editor/canvas/services.js

```diff
@@ -121,7 +127,14 @@ export async function start(canvas, options = {}) {
 }
 
 export async function stop(canvas) {
-    return API.post(`${canvasesUrl}/${canvas.id}/stop`).then(getData)
+    return API.post(`${canvasesUrl}/${canvas.id}/stop`)
+        .then((data) => {
+            Notification.push({
+                title: 'Canvas stopped.',
+                icon: NotificationIcon.CHECKMARK,
+            })
+            return getData(data)
+        })
```

Comment:
I feel like notifications should be handled at the layer above this, it's possible that canvases should be started or stopped without a notification showing. Also, this won't display a notification for historical canvases, which stop on their own. i.e. these notifications should be triggered by some UI component that monitors changes in the currently visible canvas's running state, however intent of this PR is to update notification design and having them here isn't terrible so will do for now. 👍

Reply:
Also note it doesn't show any notification for canvases that fail to start or fail to stop successfully. Most commonly they'll fail because they're already started or already stopped, in which case those errors could probably be ignored safely… but a failure for some other reason should probably be a notification. Also, if the `getData` call throws i.e. got a malformed response, the notification will still suggest success.